exporters/tokens/src/helpers/__tests__/stringHelper.test.ts

```diff
@@ -1,23 +1,43 @@
-import { toPlural } from '../stringHelper';
-
-const dataProviderItems = [
-  ['radius', 'radii'],
-  ['spacing', 'spaces'],
-  ['breakpoint', 'breakpoints'],
-  ['grid-spacing', 'grid-spacings'],
-  ['shadow', 'shadows'],
-  ['gradient', 'gradients'],
-  ['size', 'sizes'],
-  ['space', 'spaces'],
-  ['spaces', 'spaces'],
-];
+import { toPlural, toCamelCase } from '../stringHelper';
 
 describe('stringHelper', () => {
-  describe.each(dataProviderItems)('toPlural', (name, expected) => {
+  const dataProviderToPlural = [
+    ['radius', 'radii'],
+    ['spacing', 'spaces'],
+    ['breakpoint', 'breakpoints'],
+    ['grid-spacing', 'grid-spacings'],
+    ['shadow', 'shadows'],
+    ['gradient', 'gradients'],
+    ['size', 'sizes'],
+    ['space', 'spaces'],
+    ['spaces', 'spaces'],
+  ];
+
+  describe.each(dataProviderToPlural)('toPlural', (name, expected) => {
     it(`should return the expected plural form for ${name}`, () => {
       const result = toPlural(name);
 
       expect(result).toBe(expected);
     });
   });
+
+  const dataProviderToCamelCase = [
+    ['test value', 'testValue'],
+    ['another TEST val', 'anotherTestVal'],
+    ['test xsmall', 'testXSmall'],
+    ['xsmall', 'xsmall'],
+    ['xlarge', 'xlarge'],
+    ['size-xsmall', 'sizeXSmall'],
+    ['size-xlarge', 'sizeXLarge'],
+    // This will not work in this implementation: `size-xxlarge`-> `sizeXXlarge`
+    // ['size-xxlarge', 'sizeXXLarge'],
```

Comment:
How difficult is it to implement? Isn't it all about matching `xx` first and then trying `x`?

Because I think anyone can extend the scales anytime, so we should be ready for that.

Reply:
This is not a big deal, however, this is some kind of exception and @crishpeen has doubts about this change.